resource/Manifest-1.json

```diff
@@ -77,9 +74,9 @@
             }
           }
         },
-        "qooxdoo-versions": {
-          "type": "array",
-          "description": "List of qooxdoo versions this library works with. Deprecated! User requires instead."
+       "version": {
+          "type": "string",
+          "description": "ideally a semver compatible version string"
```

Comment:
I think we should be strict here : "a semver compatible string" (not "ideally"), so that nobody thinks "trunk" or "master" is an acceptable version

Reply:
and provide a pattern to validate it: https://gist.github.com/jhorsman/62eeea161a13b80e39f5249281e17c39

Reply:
added